### DISCUSSION TITLE
Storage

### DISCUSSION THREAD

**Discussion by orisa** [supabase/supabase#8746] [Questions] [answered]
How do I access my storage buckets using bash? I've tried following this swagger api 
https://supabase.github.io/storage-api/ .I get back an empty array whenever I use supabaseurl/storage/v1/bucket/
I also tried getting the bucket using supabaseurl/storage/v1/bucket/{bucketId} where bucketId is the name of the bucket in the console. Yet, I get status code 404. It seems I'm missing something. Pls help.

**Comment by GaryAustin1** [comment, marked as answer]
You have to have RLS permission on either the buckets table or the objects table depending on what you are doing.
The API guide tells you which permissions you need for each command.  https://supabase.com/docs/reference/javascript/storage-getbucket

**Reply by orisa** [reply]
Thanks. I will take a look at that.

**Reply by orisa** [reply]
Finally got it working. Thanks.
